format/view-spec.md

```diff
@@ -37,6 +37,8 @@ Each compute engine stores the metadata of the view in its proprietary format in
 
 View metadata storage mirrors how Iceberg table metadata is stored and retrieved. View metadata is maintained in metadata files. All changes to view state create a new view metadata file and completely replace the old metadata using an atomic swap. Like Iceberg tables, this atomic swap is delegated to the metastore that tracks tables and/or views by name. The view metadata file tracks the view schema, custom properties, current and past versions, as well as other metadata.
 
+When view metadata is replaced, then previously set properties are carried over.
```

Comment:
I think it's incorrect to make this statement in the spec. The "replace" operation for `REPLACE VIEW` is a SQL operation with chosen semantics that are represented in the Iceberg view metadata as you describe. But it's a very different statement to say that "when view metadata is replaced" something happens.

Saying that when metadata is "replaced" here could mean any update to the view metadata object, which is a much broader statement than you intended.

As in the table spec, we don't want to add broad statements like this. You can add a section explaining how a SQL `REPLACE VIEW` works, but even then I wouldn't say that it is a requirement of the spec.

Reply:
Removed.

Reply:
makes sense, thanks for updating this